playbook/buildfarmer/scripted_buildfarm_day.md

```diff
@@ -25,7 +25,7 @@ Check the following links and verify that each responds correctly (not showing a
 * https://build.osrfoundation.org/
 * https://ci.ros2.org/
 
-If any of the links above are not working you should check [Jenkins server restart](../infra/jenkins_server_restart.md) and report to the infra team in `#infra-core-developers` Slack channel
+If any of the links above are not working you should check with the OSRF infrastructure team to restart the Jenkins Server.
```

Comment:
`to restart the Jenkins Server.` -> `to take a look, there might be an infra problem`